src/cascadia/TerminalApp/CommandPalette.xaml

```diff
@@ -165,7 +169,7 @@ the MIT License. See LICENSE in the project root for license information. -->
                                   AutomationProperties.Name="{x:Bind Name, Mode=OneWay}"
                                   AutomationProperties.AcceleratorKey="{x:Bind KeyChordText, Mode=OneWay}">
 
-                        <Grid HorizontalAlignment="Stretch"  >
+                        <Grid HorizontalAlignment="Stretch" ColumnSpacing="5" >
```

Comment:
why 5? I feel like most things in Xaml are spaced in increments of 4. Maybe 8 would create the same "padding" on the left and right of the icon?

Reply:
5 was a random value 😄 I wasn't sure what to go with, so I'll go with 8 for now